Spigot-Server-Patches/0707-Fix-duplicating-give-items-on-item-drop-cancel.patch

```diff
@@ -0,0 +1,54 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Alphaesia <creepashadowz@gmail.com>
+Date: Fri, 23 Apr 2021 09:57:56 +1200
+Subject: [PATCH] Fix duplicating /give items on item drop cancel
+
+/give drops an unobtainable "fake item" alongside adding items to a
```

Comment:
Could the merger copy the entire PR description into this? (Keep the fixes SPIGOT stuff)